modules/cluster-logging-loki-deploy.adoc

```diff
@@ -96,23 +96,23 @@ spec:
   tenants:
     mode: openshift-logging
 ----
-<1> Or `gp2-csi`. 
-+
-.. Apply the configuration:
+<1> Or `gp2-csi`.
+
+. Apply the `LokiStack` CR:
 +
 [source,terminal]
 ----
-oc apply -f logging-loki.yaml
+$ oc apply -f logging-loki.yaml
 ----
-+
-. Create or edit a `ClusterLogging` CR:
+
+. Create a `ClusterLogging` custom resource (CR):
 +
 [source,yaml]
 ----
 apiVersion: logging.openshift.io/v1
 kind: ClusterLogging
 metadata:
-  name: instance
+  name: cr-lokistack
```

Comment:
pretty sure I saw after the fact that one or more of the CRs must be named `instance`, but I'm not sure why - note to self to check on this

Reply:
@kabirbhartiRH any thoughts on what this name should be?

Reply:
@abrennan89 The name should be 'instance' for ClusterLogging CR.